### DISCUSSION TITLE
How to train SiamMask?

### DISCUSSION THREAD

**Discussion by jiangzhengkai** [STVIR/pysot#456] [Q&A]
Hi, I want to train siammask. However, the datasets has only VOT series. Any instructions to train siammask?

**Comment by ZhiyuanChen** [comment]
> Hi, I want to train siammask. 

Glad to know. 

> However, the datasets has only VOT series. 

Yes?

> Any instructions to train siammask?

What kind of instrucitons are you looking for?
